src/http-schema/http-schema.test.ts

```diff
@@ -49,6 +53,10 @@ const enhancedArticleExample = {
         authors: [],
         pageStart: 1,
         pageEnd: 2,
+        isPartOf: {
+          type: 'PublicationVolume',
+          volumeNumber: 10738584221112336,
```

Comment:
@scottaubrey my IDE has this to say about this integer:

`TS80008: Numeric literals with absolute values equal to 2^53 or greater are too large to be represented accurately as integers.`

Then it prompts me to `Convert to a bigint numeric literal`